app/views/pages/instances/delete.liquid

```diff
@@ -0,0 +1,11 @@
+---
+method: post
```

Comment:
instead of having slug `instances/delete` and method `post`, what you want to do is to use `delete` method, and the slug should be `instances/:id` - please refer to https://documentation.platformos.com/developer-guide/modules/platformos-modules#resourceful-route-naming-convention